README.md

```diff
@@ -1,13 +1,18 @@
-voucherify-java-sdk
-===============
+Voucherify Java SDK
+===================
 
 ###Version: 2.1.0
 
-Java SDK for Voucherify API.
+[Voucherify](http://voucherify.io) has a new platform that will help your team  to automate voucher campaigns. It does this by providing composable API and the marketer-friendly interface that let growth teams:
```

Comment:
dalbym 'help your team automate' (bez to) a potem 'interface that lets', nie kumam czemu tu jest growth, voucherify pomaga rosnąć teamom?